hikari/audit_logs.py

```diff
@@ -327,18 +327,34 @@ class AuditLogEventType(int, enums.Enum):
     INTEGRATION_CREATE = 80
     INTEGRATION_UPDATE = 81
     INTEGRATION_DELETE = 82
+    STAGE_INSTANCE_CREATE = 83
+    STAGE_INSTANCE_UPDATE = 84
+    STAGE_INSTANCE_DELETE = 85
     STICKER_CREATE = 90
     STICKER_UPDATE = 91
     STICKER_DELETE = 92
     GUILD_SCHEDULED_EVENT_CREATE = 100
     GUILD_SCHEDULED_EVENT_UPDATE = 101
     GUILD_SCHEDULED_EVENT_DELETE = 102
-    APPLICATION_COMMAND_PERMISSION_UPDATE = 121
     THREAD_CREATE = 110
     THREAD_UPDATE = 111
     THREAD_DELETE = 112
+    APPLICATION_COMMAND_PERMISSION_UPDATE = 121
+    AUTO_MODERATION_RULE_CREATE = 140
+    AUTO_MODERATION_RULE_UPDATE = 141
+    AUTO_MODERATION_RULE_DELETE = 142
+    AUTO_MODERATION_BLOCK_MESSAGE = 143
+    AUTO_MODERATION_FLAG_TO_CHANNEL = 144
+    AUTO_MODERATION_USER_COMMUNICATION_DISABLED = 145
```

Comment:
These are addressed in the automod PR (which I might be looking into redoing soon)